iocore/eventsystem/UnixEThread.cc

```diff
@@ -41,27 +41,12 @@ struct AIOCallback;
 
 bool shutdown_event_system = false;
 
-EThread::EThread()
-  : generator((uint64_t)Thread::get_hrtime_updated() ^ (uint64_t)(uintptr_t)this),
-    ethreads_to_be_signalled(nullptr),
-    n_ethreads_to_be_signalled(0),
-    id(NO_ETHREAD_ID),
-    event_types(0),
-    signal_hook(nullptr),
-    tt(REGULAR)
+EThread::EThread() : id(NO_ETHREAD_ID), tt(REGULAR)
 {
   memset(thread_private, 0, PER_THREAD_DATA);
 }
 
-EThread::EThread(ThreadType att, int anid)
-  : generator((uint64_t)Thread::get_hrtime_updated() ^ (uint64_t)(uintptr_t)this),
-    ethreads_to_be_signalled(nullptr),
-    n_ethreads_to_be_signalled(0),
-    id(anid),
-    event_types(0),
-    signal_hook(nullptr),
-    tt(att),
-    server_session_pool(nullptr)
+EThread::EThread(ThreadType att, int anid) : id(anid), tt(att), server_session_pool(NULL)
```

Comment:
Because this was done initially before the Great Null Shift and I missed this during the merge.